pkgs/by-name/xa/xavs2/package.nix

```diff
@@ -0,0 +1,70 @@
+{
+  lib,
+  fetchFromGitHub,
+  gitUpdater,
+  nasm,
+  stdenv,
+  testers,
+  validatePkgConfig,
+  pkgsBuildBuild,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "xavs2";
+  version = "1.4";
+
+  src = fetchFromGitHub {
+    owner = "pkuvcl";
+    repo = "xavs2";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-4w/WTXvRQbohKL+YALQCCYglHQKVvehlUYfitX/lLPw=";
+  };
+
+  env.NIX_CFLAGS_COMPILE = toString [
+    # Fixes compile error on darwin
+    "-Wno-incompatible-function-pointer-types"
+  ];
+
+  preConfigure = ''
+    # Generate version.h
+    ./version.sh
+    cd build/linux
+    export AS=nasm
+  '';
+
+  configureFlags = lib.optionals (!stdenv.hostPlatform.isStatic) [
+    (lib.enableFeature true "shared")
+    "--system-libxavs2"
+  ];
+
+  postInstall = lib.optionalString (!stdenv.hostPlatform.isStatic) ''
+    rm $lib/lib/*.a
+  '';
```

Comment:
Do we normally delete static libraries entirely like this? I thought `isStatic` was about whether executables were linked statically.

Reply:
I thought it was about both static libraries and executables.